testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -730,7 +742,25 @@ impl BitcoinRegtestController {
     ) -> Option<Transaction> {
         let public_key = signer.get_public_key();
 
-        let (mut tx, utxos) = self.prepare_tx(&public_key, payload.burn_fee, attempt)?;
+        let sunset_fee = if payload.sunset_burn > 0 {
+            cmp::max(payload.sunset_burn, DUST_UTXO_LIMIT)
+        } else {
+            0
+        };
+
+        let number_of_transfers = payload.commit_outs.len() as u64;
+        let value_per_transfer = payload.burn_fee / number_of_transfers;
+
+        let btc_miner_fee = self.config.burnchain.block_commit_tx_estimated_size
+            * self.config.burnchain.satoshis_per_byte
+            * attempt;
```

Comment:
Is the `* attempt` at the end increasing the fee needlessly, considering that there's a separate `rbf_fee`?

Reply:
ha good catch, I've been doing a few back and forth there, thanks.
Addressed with https://github.com/blockstack/stacks-blockchain/pull/2317/commits/46c89a2f299a5593d5137fd25a8a210f6035dd1c